app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    split_command = command.split(" ")
+    if split_command[1] != split_command[2]:
```

Comment:
Check that length of split result is exactly 3
Check that first element is 'cp'